streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamWindowAggregate.java

```diff
@@ -59,6 +59,10 @@
         return new KStreamWindowAggregateProcessor();
     }
 
+    public Windows<W> windows() {
```

Comment:
Why not `TimeWindows`?

Reply:
How do you know it's a `TimeWindows`?

Reply:
Sorry, I was being silly, but upon second reading, it looks snarky... `Windows<W>` is the tightest bound we can put on the window spec at this point, since this processor takes any window spec.

Reply:
I think `KStreamWindowAggregate` is used for any subclass of `Windows`, of which `TimeWindows` is one.

It's definitely not `SessionWindows`, since `SessionWindows` is not a subclass of `Windows`, but it could be `UnlimitedWindows` or any user-supplied `Windows` subclass.